### PR TITLE
pulumi-install: add page

### PR DESCRIPTION
**Pull request by Denis-Kuso**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by gutjuri — APPROVED** [review]

File: pages/common/pulumi-install.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# pulumi install
+
+> Install packages and plugins required by your existing project.
```
Comment (spageektti) [inline pages/common/pulumi-install.md:3]:
```suggestion
> Install packages and plugins for the current program or policy pack.
```  
What about using the same description as in the documentation?

Comment (Denis-Kuso) [reply pages/common/pulumi-install.md:3]: I reworded it to highlight that a project needs to exist, else the command fails. I guess, the word "current" kind of implies existence.

File: pages/common/pulumi-install.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# pulumi install
+
+> Install packages and plugins for the current program or policy pack.
+> More information: <https://www.pulumi.com/docs/iac/cli/commands/pulumi_install>.
+
```
Comment (nelsonfigueroa) [inline pages/common/pulumi-install.md:5]:
```suggestion

- Install packages and plugins:

`pulumi install`

```
`pulumi install` works as-is, so I think we should have the base command listed.

Comment (nelsonfigueroa) [reply pages/common/pulumi-install.md:5]: Looks good to me aside from this suggestion though!

**Review by nelsonfigueroa — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

**Pull request merged**